templates/vm/votp_vm_standard_config.xml.j2

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (C) 2021, RTE (http://www.rte-france.com) -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<domain type="{{ vm_domaine | default('kvm') }}">
```

Comment:
vm_domain?

Reply:
Nope, based on other VM configuration, it is `vm_domaine`